api/qupo_backend/tickers_api.py

```diff
@@ -0,0 +1,105 @@
+import logging
+
+from fastapi import APIRouter, Depends, HTTPException
+from pytickersymbols import PyTickerSymbols
+from sqlalchemy.orm import Session, sessionmaker
+
+from config import settings
+from .db import models, schemas
+from .db.database import SessionLocal, engine
+from .integrator import get_all_symbols, get_data_of_symbol
+
+
+router = APIRouter(
+    prefix='/tickers',
+    tags=['tickers'],
+)
+
+settings = settings()
+stock_data = PyTickerSymbols()
+
+
+def get_db():
```

Comment:
this function I would maybe put into the database.py module, it will be commonly reused by the rest of the code or?